lib/Sources/BluetoothMessage/BluetoothState.swift

```diff
@@ -56,18 +64,26 @@ public actor BluetoothState {
         return peripheral
     }
 
-    public func rememberPeripheral(_ uuid: UUID) {
-        // TODO: persist known peripheral identifier
+    public func rememberPeripheral(identifier uuid: UUID) async {
+        var peripheralIds = await getKnownPeripheralIdentifiers()
+        peripheralIds.insert(uuid)
+        await save(peripheralIds: peripheralIds)
     }
 
-    public func forgetPeripheral(_ uuid: UUID) {
-        self.peripherals.removeValue(forKey: uuid)
-        // TODO: un-persist known peripheral identifier
+    public func forgetPeripheral(identifier uuid: UUID) async {
+        var peripheralIds = await getKnownPeripheralIdentifiers()
+        peripheralIds.remove(uuid)
+        await save(peripheralIds: peripheralIds)
+    }
+
+    public func forgetPeripherals(identifiers: [UUID]) async {
+        for uuidToForget in identifiers {
+            await forgetPeripheral(identifier: uuidToForget)
+        }
```

Comment:
nit: more efficient to do a single read/deserialize/alter/reserialize/write pass instead of n:
```suggestion
        var peripheralIds = await getKnownPeripheralIdentifiers()
        identifiers.forEach { peripheralIds.remove($0) }
        await save(peripheralIds: peripheralIds)
```

Is a nit because the write is debounced anyhow, so I don't mind either way.